pkg/sql/plan/base_binder.go

```diff
@@ -851,6 +851,11 @@ func bindFuncExprImplByPlanExpr(name string, args []*Expr) (*plan.Expr, error) {
 		}
 	}
 
+	if name == "uuid" {
```

Comment:
Do we have a way to get this magic from function id?   Or even better, let's give all these functions a flag and check for the flag.   For example, how about nextval of sequence and a argumentless random function?

Reply:
OK, I'll try it